<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SupplierHomeWindowN</class>
 <widget class="QMainWindow" name="SupplierHomeWindowN">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="vendorLabel">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>533</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>781</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Добавить товар</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Название</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Артикул</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>220</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Цена</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>340</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_13">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Количество</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Изменить товар</string>
     </attribute>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Выберите товар</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Название</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>220</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Артикул</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>260</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Цена</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changeButton">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>320</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Change</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>140</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Новое название</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Новый артикул</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_12">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>260</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Новая цена</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_14">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Количество</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_15">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>180</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Новое количество</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Удалить товар</string>
     </attribute>
     <widget class="QComboBox" name="comboBox_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Выберите товар</string>
      </property>
     </widget>
     <widget class="QPushButton" name="deleteButton">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>340</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Название</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>220</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Артикул</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>260</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Цена</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_16">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Количество</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
